docker-compose.yml

```diff
@@ -123,7 +123,7 @@ services:
             - MINIO_ACCESS_KEY=minio
             - MINIO_SECRET_KEY=minio123
         volumes:
-            - '${PWD}:/export'
+            - '${MEDIADIRECTORY}:/export'
```

Comment:
The Minio instance is designed to be a web interface to ~all~ of the backend folders of the project.
So changing `$PWD` to `${MEDIADIRECTORY}` doesn't really get us the same thing.
Looking at Minio you have to spin up separate containers for different top level directories so if we added this we would have to include a second Minio to account for access to `${DATADIRECTORY}`.